themes/socialblue/color/color.inc

```diff
@@ -5,6 +5,16 @@
  * Lists available colors and color schemes for the Social Blue theme.
  */
 
+$brand_css = [
+  'assets/css/brand.css',
+];
+// Add brand css files for new Sky style.
+$theme_path = drupal_get_path('theme', 'socialblue');
+$sky_brand_css = 'assets/css/brand--sky.css';
+if (file_exists($theme_path . '/' . $sky_brand_css)) {
```

Comment:
Isn't this always true?

Reply:
Good catch - I simplified it and fixed in PR #1680